### ISSUE TITLE
Firefox extension 2.1 disabled for XDM 7.2.11

### ISSUE THREAD

**Issue by Ma6nu5** [subhra74/xdm#194]
When I install 7.2.11, the Firefox extension 2.1 stops working. Falling back to 7.2.10 and it works fine. 

I had problems with 7.2.11 also with the Chrome extension, but that could be solved just by reinstalling the extension. This circumvention did not work in Firefox, the extension loads OK, then disables itself after around five seconds. 

Also, installing the Firefox extension from XDM 7.2.11 Tools > Options > Browser monitoring doesn't work for me, the button turns blue and then nothing happens.

Windows 10.1903, Firefox 76.0.1

**Comment by subhra74** [comment]
Please generate the logs mentioned in #197

**Comment by Ma6nu5** [comment]
I installed 7.2.11 again and the extension was immediately disabled, so the problem persists. Enclosing the log, please let me know if I did anything wrong when saving it to a file.
[console-export-2020-7-16_7-17-49.txt](https://github.com/subhra74/xdm/files/4929472/console-export-2020-7-16_7-17-49.txt)

**Comment by subhra74** [comment]
> I installed 7.2.11 again and the extension was immediately disabled, so the problem persists. Enclosing the log, please let me know if I did anything wrong when saving it to a file.
> [console-export-2020-7-16_7-17-49.txt](https://github.com/subhra74/xdm/files/4929472/console-export-2020-7-16_7-17-49.txt)

Thanks for sharing the log, it looks like connection to loopback is failing.
Take a look at #235 it might help

**Comment by Ma6nu5** [comment]
![privacy-firstparty-isolate](https://user-images.githubusercontent.com/37984620/88019949-12f06c80-cb2b-11ea-9a65-30dd5be99883.png)

If you mean the privacy setting mentioned in #235 it is already set to false in my case, so unfortunately this will not solve my problem. Please note that it failed also with Chrome. 7.2.10 works fine for me, and I have reinstalled it several times without encountering any problem with any of the browser add-ins, so this issue is specifically related to 7.2.11 that never works correctly for me with any browser add-in. If no-one else has this problem you can ignore it and I will let you know how future updates work for me.

**Comment by joelbudgor** [comment]
I am having exactly the same problem in Firefox 92.

Sorry to report but this problem still exists.

Checked my setting and privacy.firstparty.isolate already set to false.

When I start Firefox I get s screen prompting me to agree to the XDM privacy policy.

I accept and the XDM icon is immediately showing up as disabled in Firefox.

Have had this problem for at least a year, to the point where I gave up and stopped using the plugin entirely.

Tried again recently, but as described above, to no avail.

No VPN involved.
